cluster_info/dodrio-gpu-rome-a100-40.json

```diff
@@ -0,0 +1,13 @@
+{
+    "qdel_cmd_tpl": "scancel --cluster=dodrio {{ cluster_job_id }}",
+    "worker_bin_path" : "__worker_path__/rocky8_icelake/cryosparc_worker/bin/cryosparcw",
+    "title": "dodrio-gpu-rome-a100-40",
+    "cache_path": "/ssd/tmp",
+    "qinfo_cmd_tpl": "sinfo --cluster=dodrio --format='%.8N %.6D %.10P %.6T %.14C %.5c %.6z %.7m %.7G %.9d %20E'",
+    "qsub_cmd_tpl": "sbatch --parsable --cluster=dodrio {{ script_path_abs }} 2> {{ script_path_abs }}.submitfilter | cut -d ';' -f1",
```

Comment:
*Probably* you will need to add `SBATCH_PARTITION=gpu_rome_a100` in front of the `sbatch` command, otherwise you will submit to the default `cpu_rome` partition of Hortense.

Reply:
fixed in 1294fcd.
Can you please give it a shot on Hortense to be sure that the last proposed change targets the correct partition?

Reply:
I started my own branch here https://github.com/hpcleuven/cryoSPARC-VSC/pull/3 and with that I got things working on Hortense. Currently I am waiting for feedback from Tatiana, who will test this after the holidays.

Reply:
I see how nicely the PR #3 complements this one. My intention is to not to merge `build4hortense` into the `master` branch, but rather keeping it as a stand-alone branch.
So, I though that your branch called `hortense` might need to merge into `build4hortense` (or the other way around). 
What do you think of this.

Reply:
My intention with the `hortense` branch is to merge it into `main`. Although there are some specifics to each cluster, I think it would be easier to have a single branch that provides instructions that work both on Hortense and on KU Leuven Tier-2. Where it is necessary to make a distinction (for example to take partitions available on each cluster), I just work with the `VSC_INSTITUTE_CLUSTER` variable to keep scripts generally applicable (on VSC clusters only of course).